zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/user/UserRecord.java

```diff
@@ -17,10 +17,10 @@
 import org.agrona.DirectBuffer;
 
 public final class UserRecord extends UnifiedRecordValue implements UserRecordValue {
-  private final StringProperty usernameProp = new StringProperty("username");
-  private final StringProperty nameProp = new StringProperty("name");
-  private final StringProperty emailProp = new StringProperty("email");
-  private final StringProperty passwordProp = new StringProperty("password");
+  private final StringProperty usernameProp = new StringProperty("username", "");
```

Comment:
❌ I don't think username should be optional. As-is we can't do anything without having a username set, as it's the current identifier for a user.